R/utilities.R

```diff
@@ -471,11 +472,11 @@ update_SE_from_tibble <- function(.data_mutated, se, column_belonging = NULL) {
                 
                 # Pivot for generalising to many assays
                 pivot_longer(cols = -c(!!s_(se)$symbol, !!f_(se)$symbol)) %>%
-                nest(data___ = c(!!s_(se)$symbol, !!f_(se)$symbol, value)) %>%
+                nest(data___ = c(!!s_(se)$symbol, !!f_(se)$symbol, .data$value)) %>%
```

Comment:
the tidy evaluation works without the `.data$` Any specific reason you add all those?